[core]/es_extended/server/commands.lua

```diff
@@ -574,11 +574,11 @@ ESX.RegisterCommand("noclip", 'admin', function(xPlayer)
 end, false)
 
 ESX.RegisterCommand('players', "admin", function()
-	local xPlayers = ESX.GetExtendedPlayers() -- Returns all xPlayers
-	print(('^5%s^2 online player(s)^0'):format(#xPlayers))
-	for i = 1, #(xPlayers) do
-		local xPlayer = xPlayers[i]
-		print(('^1[^2ID: ^5%s^0 | ^2Name : ^5%s^0 | ^2Group : ^5%s^0 | ^2Identifier : ^5%s^1]^0\n'):format(
-			xPlayer.source, xPlayer.getName(), xPlayer.getGroup(), xPlayer.identifier))
-	end
+    local xPlayers = ESX.GetExtendedPlayers() -- Returns all xPlayers
+    local count = 0 
+    for _, xPlayer in pairs(xPlayers) do
```

Comment:
You are wrong, the counting would not be interrupted because pairs are used instead of ipairs.
I do not consider this PR relevant

Reply:
> You are wrong, the counting would not be interrupted because `pairs` are used instead of `ipairs`. I do not consider this PR relevant

What do you mean? 
```lua
print(('^5%s^2 online player(s)^0'):format(#xPlayers))
```
Would result in wrong output as # is not nil value safe.

Reply:
> 

i was talking about the code before, #xPlayers would break when someone leaves.

Reply:
sorry, I confused the old version with the new one